Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1

```diff
@@ -0,0 +1,275 @@
+$ProjectGuidAndFilePathMap = @{}
```

Comment:
Most of the code here already existed and was reviewed. I made small refactorings to better structure it, but no functional changes.